mod_lookup_identity.c

```diff
@@ -88,6 +119,11 @@ static int lookup_user_by_certificate(request_rec * r) {
 	dbus_error_init(&error);
 	DBusMessage * message = NULL;
 	DBusMessage * reply = NULL;
+	char * username;
+
+	if ((username = get_value_from_query(r, "username")) == NULL) {
```

Comment:
The name of the value holding the username in the query string should be configurable.